ORM/NullableEmbeddableListener.php

```diff
@@ -16,9 +16,10 @@
     private $propertyMap = [];
     private $useClosure = false;
 
-    public function __construct(PropertyAccessorInterface $propertyAccessor)
+    public function __construct(PropertyAccessorInterface $propertyAccessor, bool $useClosure = false)
     {
         $this->propertyAccessor = $propertyAccessor;
+        $this->useClosure = false;
```

Comment:
This should be `$this->useClosure = $useClosure`, shouldn't it?

Reply:
Oh x) . Yeah, of course.